### ISSUE TITLE
Who supports this?

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/change-password-url#16]
Hi, WICG chairs are just going through the repos and checking status... I seem to recall Safari supports this feature (and twitter was using it, right?)? Did other browser vendors get behind it? If yes, should we consider moving it to the W3C or WHATWG?

**Comment by tschoffelen** [comment]
I think there's quite a lot of websites that implement this now (including Facebook and Twitter), and a few password managers and browsers make use of this spec (Safari, 1Password).

**Comment by marcoscaceres** [comment]
Great to hear that big sites are using it. What about other browsers?

**Comment by battre** [comment]
Chrome is generally interested in this, but we have some concerns about the reliability that we haven't quantified, yet.

We see for example servers responding with "200 OK" and a page that says "this page does not exist" or that does not have a login form when you are logged out. I would consider to add a mandatory header that says "Server understands this spec, my HTTP response code is reliable".

To pick on ourselves instead of others: https://news.google.com/.well-known/change-password redirects me to https://news.google.com/?hl=de&gl=DE&ceid=DE:de.

Also there is no commitment on any timeline from our side, yet.

**Comment by tschoffelen** [comment]
@battre Yep, that's something that others realised as well: https://github.com/WICG/change-password-url/issues/14

**Comment by hober** [comment]
@battre wrote:

> Chrome is generally interested in this, but we have some concerns about the reliability that we haven't quantified, yet.
> 
> We see for example servers responding with "200 OK" and a page that says "this page does not exist" or that does not have a login form when you are logged out.

Yeah, we've seen this too, and like @tschoffelen points out, it came up before in #14. In that issue, we concluded that a separate well-known URL would be best for detecting this, and in 0b98642 I've defined such a mechanism in [a separate document in this repository](https://wicg.github.io/change-password-url/response-code-reliability.html). Please let me know what you think.

**Comment by hober** [comment]
@battre, any thoughts?

**Comment by battre** [comment]
The proposal in https://wicg.github.io/change-password-url/response-code-reliability.html makes sense to me and this is what we are currently implementing.

**Comment by marcoscaceres** [comment]
Thanks for the update @battre ... seeing if there is interest from the Gecko side.

**Comment by marcoscaceres** [comment]
Filed https://github.com/mozilla/standards-positions/issues/372 for a Mozilla position.

**Comment by FaRrules** [comment]
This is used against me

**Comment by FaRrules** [comment]
Is it spam?

**Comment by FaRrules** [comment]
I dont have moszilla

**Comment by FaRrules** [comment]
Bounty hunting

**Comment by FaRrules** [comment]
Guys remember me
